.github/workflows/lint-and-test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           corepack enable
           corepack prepare pnpm@latest --activate
           pnpm install
```

Comment:
```suggestion
          pnpm ci
```

Reply:
🔗 https://pnpm.io/next/continuous-integration#github-actions

they provide a recipe for ci (even this one), but it doesn't seem better that what we currently have